lib/routes/account.js

```diff
@@ -512,6 +512,9 @@ module.exports = function (
                     email: email,
                     errno: err.errno
                   })
+                  if (customsErr) {
+                    throw customsErr
+                  }
```

Comment:
Should this be inside or outside the ` if (err.errno === error.ERRNO.ACCOUNT_UNKNOWN)`?  I can see an argument that it should be outside, so that the behavior is indistinguishable from what it would have been if unblock wasn't in play.